### ISSUE TITLE
Manage WRC digests

### ISSUE THREAD

**Issue by viroulep** [thewca/worldcubeassociation.org#2539]
Right now we manage WRC digests manually.

The incidents log will have the incidents dates, if they are digest-worthy, and if they are resolved.
We could automatically draft a digest based on which digest-worthy incident were resolved since last digest, including the public summary of the incident.
And also send the draft directly from the website when we're happy with it.

**Comment by AlbertoPdRF** [comment]
And also we need to be able to mark the digest worthy incidents as sent.

**Comment by AlbertoPdRF** [comment]
This actually exists :P Then I think the only necessary thing is to display it on the main menu, similar as how the "Status" is shown.

**Comment by viroulep** [comment]
I handled this in #2802.

**Comment by jfly** [comment]
@viroulep, can this issue be closed now? Thanks!

**Comment by viroulep** [comment]
Not really, I handled Alberto's comment but the original issue still exists: we could have an automatically generated digest draft from the incidents information!
